ios/MaasNearbyTransit/Sources/MaasNearbyTransit/Components/NearbyTransitStopBadgeView.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+import Swappable
+
+struct NearbyTransitStopBadgeView: View, Swappable {
+
+    var constants: Kotlin<NearbyTransitStopBadgeConstants> { Kotlin(NearbyTransitStopBadgeConstants()) }
+
+    public struct InputType {
+        public let schedules: [Schedule]
+    }
+    let input: InputType
+
+    public init(input: InputType) {
+        self.input = input
+    }
+
+    public init(schedules: [Schedule]) {
+        self.input = .init(schedules: schedules)
+    }
+
+    private var slicedFilteredSchedules: [Schedule] {
+        input.schedules.filterDuplicates(includeElement: { $0.transportId == $1.transportId })
+    }
+
+    var defaultBody: some View {
+        ZStack {
+            ForEach(slicedFilteredSchedules.indices) {
+                imageView(schedule: slicedFilteredSchedules[$0])
+                    .padding([.leading, .bottom], $0 != 0 ? CGFloat((-constants.offsetXY * CGFloat($0))) : 0)
```

Comment:
It's hard to understand..
could refactor to `func paddingForItem(at: Int) -> CGFloat`

Reply:
Sure!